src/iva-app.html

```diff
@@ -714,8 +714,36 @@ <h3>Loading...</h3>
                     this.tool = "#home";
                 }
                 this.renderHashFragments();
-
                 this.getProjects();
+                // 60000 ms = 1 min. Every 1 min we check if session is close to expirate.
+                this.intervalCheckSession = setInterval(this.checkSessionActive.bind(this), 60000);
+            }
+
+            checkSessionActive() {
+                let _message = "";
+                let _options = {
+                    icon: 'glyphicon glyphicon-user',
+                };
+                if (this._logged) {
+                    let decoded = jwt_decode(this.sessionId);
+                    let currentTime = new Date().getTime();
+                    let remainingTime = ((decoded.exp * 1000) - currentTime);
+                    // 300000 ms = 5 min = 1000(1sec) * 60(60 sec = 1min) * 5(5 min)
+                    if (remainingTime <= 300000 && remainingTime >= 60000) {
```

Comment:
I think it would be a good idea to move this numbers to config file, probably 'application'